cmd/migrate/migrate.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/ledgerwatch/bolt"
+	"github.com/ledgerwatch/turbo-geth/common/dbutils"
+)
+
+func encodeToCBOR(buf []byte) ([]byte, error) {
+	acc, err := decodeRLP(buf)
+
+	if err != nil {
+		return nil, err
+	}
+
+	res := make([]byte, acc.EncodingLengthForStorage())
+	acc.EncodeForStorage(res)
+	return res, nil
+}
+
+// This method does not Commit
+func convertDatabaseToCBOR(db *bolt.DB, maxTxOperations uint) error {
+	var k, v []byte
+	var done bool = false
+	for !done {
+		var i uint
+		err := db.Update(func(tx *bolt.Tx) error {
+			var accountBucket *bolt.Bucket = tx.Bucket(dbutils.AccountsBucket)
+			c := accountBucket.Cursor()
+
+			for k, v = c.Seek(k); k != nil; k, v = c.Next() {
```

Comment:
Looks like `c.Seek(k)` does not work very well when `k==nil`. So I suggest this:
````
if k == nil {
   k, v = c.First()
} else {
   k, v = c.Seek(k)
   k, v = c.Next()
}
````
The last call to `c.Next()` is to skip the record that has already been processed